src/Auth/AuthContainer.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+// import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import Header from 'grommet/components/Header';
+
+function AuthContainer({ children }) {
```

Comment:
`` export function AuthContainer({ children }) { `` makes it easier to test than `` function AuthContainer({ children }) { ``. What do you think?